bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -345,7 +347,7 @@ public static String getAppVersion() {
      */
     @Nullable
     public static String[] getNotifyReleaseStages() {
-        return getClient().getConfig().getNotifyReleaseStages();
+        return getClient().getConfig().getNotifyReleaseStages().toArray(new String[]{});
```

Comment:
is this an array by necessity of the way items are passed from JVM->native?

Reply:
It's been kept as an array as this will reduce rework required in the NDK/Unity, and as we're creating a copy of an array rather than exposing the original, the caller won't be able to mutate this state.